app/styleguide/core/grid/grid.md

```diff
@@ -0,0 +1,32 @@
+### Available breakpoints
```

Comment:
- [ ] Did you merge the current `develop` branch? I think these changes are from the merged variable branch and therefore should not show up here.